src/app/project-dialog/project-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="col-md-12 col-sm-12 col-xs-12 createProjectBlock">
+    <div class="col-md-12 col-sm-12 col-xs-12">
+        <h4 class="projectname">Project Name</h4>
+        <form role="form" class="search-pf has-button">
+            <div class="form-group has-clear">
+                <div class="search-pf-input-group">
+                    <input id="projectname" type="search" class="form-control" placeholder="Enter a unique name for your project">
+                </div>
+            </div>
+            <div class="form-group">
+                <button class="btn btn-default" type="button"><span class="caret pull-right icon-caret"></span></button>
+            </div>
```

Comment:
We can go with the input field only. When creating a project, we will not be searching and selecting a name.